fontes/lexador/lexador.ts

```diff
@@ -210,15 +214,30 @@ export class Lexador implements LexadorInterface {
         this.adicionarSimbolo(tiposDeSimbolos.TEXTO, valor);
     }
 
+    identificarVariaveis(): boolean {
+        if (this.simbolos.length !== 0) {
+            const indexAnterior: number = this.simbolos.length - 1;
+            if (this.simbolos[indexAnterior].tipo === tiposDeSimbolos.CIFRAO) {
```

Comment:
Substituir por `return this.simbolos[indexAnterior].tipo === tiposDeSimbolos.CIFRAO;`